externs/olx.js

```diff
@@ -4361,7 +4378,6 @@ olx.source.TileImageOptions.prototype.wrapX;
  *            cacheSize: (number|undefined),
  *            format: (ol.format.Feature|undefined),
  *            logo: (string|olx.LogoOptions|undefined),
- *            opaque: (boolean|undefined),
```

Comment:
True, it will not break applications. But I think it would be good if we document when options are removed from the API.